sklearn/multioutput.py

```diff
@@ -404,8 +404,19 @@ def fit(self, X, Y, sample_weight=None, **fit_params):
         self.classes_ = [estimator.classes_ for estimator in self.estimators_]
         return self
 
-    @property
-    def predict_proba(self):
+    def _check_predict_proba(self):
+        if hasattr(self, "estimators_"):
```

Comment:
@harupy I propagate a change to be consistent with other PRs where we decided to delegate the error message to `geattr`.

Reply:
Thanks!